README.md

```diff
@@ -1,14 +1,38 @@
 # freegeek-info-script
-Shell script for getting info on Linux Mint needed for Freegeek build sheets
+Shell script for Linux Mint that conveniently pools together info needed for Free Geek build sheets.
 
-How to run:
+### How to run:
+1. Type `Ctrl + Alt + T` to open a terminal
+2. Paste `git clone https://github.com/Owen-sz/freegeek-info-script.git` into the terminal to clone (download) the script
```

Comment:
Make these commands a separate line, like in my current README, then there is a copy button and can have less text while being easier to scroll to quickly to find the commands.